File: Iniciante/URI 1012.hs
import Text.Printf

main = do
    line <- getLine

    let (a:b:c:_) = map (read :: String -> Double) $ words line
    let triangulo = a * c / 2.0 :: Double
    let circulo = 3.14159 * (c * c) :: Double
    let trapezio = ((a + b) * c) / 2.0 :: Double
    let quadrado = b * b :: Double
    let retangulo = a * b :: Double

    putStrLn $ printf "TRIANGULO: %.3f" triangulo
    putStrLn $ printf "CIRCULO: %.3f" circulo
    putStrLn $ printf "TRAPEZIO: %.3f" trapezio
    putStrLn $ printf "QUADRADO: %.3f" quadrado
    putStrLn $ printf "RETANGULO: %.3f" retangulo

File: Iniciante/URI 1012.pas
var
	a : double;
	b : double;
	c : double;
	triangulo : double;
	circulo : double;
	trapezio : double;
	quadrado : double;
	retangulo : double;

begin
    readLn(a, b, c);
	
	triangulo := a * c / 2.0;
	circulo := 3.14159 * (c * c);
	trapezio := ((a + b) * c) / 2.0;
	quadrado := b * b;
	retangulo := a * b;
    
    writeln('TRIANGULO: ', triangulo:0:3);
    writeln('CIRCULO: ', circulo:0:3);
    writeln('TRAPEZIO: ', trapezio:0:3);
    writeln('QUADRADO: ', quadrado:0:3);
    writeln('RETANGULO: ', retangulo:0:3);
end.